### ISSUE TITLE
brew install failyer when -g for crowrap

### ISSUE THREAD

**Issue by tomfuertes** [CROmetrics/crometrics#1]
I think the issues is that this line

https://github.com/CROmetrics/crometrics/blob/master/bin/postinstall.js#L19

replaces all instances of "lib" and/or the first which is unfortunate b/c brew path's contain lib further up the chain ala:

![image](https://cloud.githubusercontent.com/assets/1503991/15873959/95c4c226-2cc6-11e6-9d6c-ddb5f195a45c.png)

**Comment by tomfuertes** [comment]
cc @casecode to subscribe in case you want to merge when lands so you can `npm install -g clearhead`

**Comment by tomfuertes** [comment]
test case
```js
$ node
> var tmpString = '/usr/loca/bin/lib/node_modules/crowrap/node_modules/crometrics/lib/';
> assert.true(doNewReplace(tmpString) === '/usr/loca/bin/lib/node_modules/crowrap/node_modules/crometrics/');
```

**Comment by tomfuertes** [comment]
Took a stab - trying to build / build / deploy

**Comment by tomfuertes** [comment]
https://github.com/CROmetrics/crometrics/commit/747547c7a173dfcb62ec0d17567c20517514d74e

**Comment by tomfuertes** [comment]
Fixed in 0.1.5
